notebooks/non-sequential.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>I wonder if it would be possible to plot using the sktime plotting utilities?  could make plotting look more uniform across the tutorial. If not would be nice improving the look/readability of this graph. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I use the sktime utilities now, thank you for the hint